src/React.js

```diff
@@ -60,7 +61,13 @@ function createClass(baseClass) {
   };
 }
 
-function createClassWithDerivedState(classCtr) {
+var componentImpl = createClass(React.Component);
+exports.componentImpl = componentImpl;
+
+var pureComponentImpl = createClass(React.PureComponent);
+exports.pureComponentImpl = pureComponentImpl;
+
+function createClassWithDerivedState() {
```

Comment:
I removed the `classCtr` argument from this function (see the diff) because it's not used in the body of the function at all. Please let me know if this needs to be re-added or handled differently.